acrobat/scripts/scripts.js

```diff
@@ -19,6 +21,17 @@ function loadStyles(paths) {
   });
 }
 
+function addLocale() {
+  const url = new URL(window.location.href);
+  const langFromPath = url.pathname.split('/')[1];
```

Comment:
```suggestion
  const { pathname } = window.location;
  const langFromPath = pathname.split('/')[1];
```
This is simpler way to get pathname from location.